elasticsearch.yml

```diff
@@ -1,8 +1,11 @@
+network.host: 0.0.0.0
+
 cluster:
   name: elasticsearch
 
 script:
-  disable_dynamic: false
+  inline: on
+  indexed: on
 
```

Comment:
We are using the new settings format in ES 1.x.  My initial testing shows that ES starts up and works with these settings, but I don't know if scripts actually work because we don't use any scripts.
We could instead have two separate files: elasticsearch-1.x.yaml and elasticsearch-2.x.yaml - Dockerfile would copy elasticsearch-1.x.yaml to elasticsearch.yml and Dockerfile-2.x would copy elasticsearch-2.x.yaml to elasticsearch.yml - but that will complicate ES maintenance and integration with other projects.